src/Components/Products/Products.css

```diff
@@ -35,24 +35,69 @@
     border-radius: 4px;   
 }
 
-@media screen and (max-width: 767px) {
+@media (max-width: 1024px) {
     .products__col {
         display: flex;
         flex-direction: column;
         min-height: 230px;
-        width: 48%;
+        width: 30.8%;
         margin-bottom: 20px;
         border-style: solid;
         border-width: 1px;
         border-color: rgb(235, 227, 227);
         border-radius: 4px;   
     }
 
+    .products__row {
+        display: flex;
+        flex-wrap: wrap;
+        max-width: 1300px;
+        justify-content: space-around;
+        box-sizing: border-box;
+        padding: 0px 10px;
+    }
+}
+
+@media screen and (max-width: 767px) {
+    .products__col {
+        display: flex;
+        flex-direction: column;
+        min-height: 230px;
+        width: 46.71%;
+        margin-bottom: 10px;
+        margin-right: 10px;
+        border-style: solid;
+        border-width: 1px;
+        border-color: rgb(235, 227, 227);
+        border-radius: 4px;   
+    }
+
     .products__row {
         display: flex;
         flex-wrap: wrap;
         padding-left: 10px;
-        padding-right: 10px;
+        padding-right: 0px;
+        width: 1300px;
+        justify-content: space-around;
+    }
+}
+
+@media (max-width: 374px) {
+    .products__col {
+        display: flex;
+        flex-direction: column;
+        min-height: 230px;
+        width: 65%;
```

Comment:
width must be 100%;
you should limit width with container

Reply:
Fixed.